.travis.yml

```diff
@@ -1,11 +1,38 @@
 language: go
 sudo: false
 go:
-  - 1.6
-  - 1.7
+  #- 1.6
+  #- 1.7
   - 1.8
-  - tip
+  #- tip
 matrix:
   allow_failures:
     - go: tip
   fast_finish: true
+sudo: false
+
+env:
+  global:
+    - STRIPE_STUB_PORT=6065
+    - STRIPE_STUB_VERSION=0.1.1
+
+cache:
+  directories:
+    - stripestubs
+
+before_install:
+  - go get -u github.com/stretchr/testify/require
+
+  # Unpack and start the Stripe API stub so that the test suite can talk to it
+  - |
+    if [ ! -d "stripestubs/stripestub_${STRIPE_STUB_VERSION}" ]; then
+      mkdir -p stripestubs/stripestub_${STRIPE_STUB_VERSION}/
+      curl -L "https://github.com/brandur/stripestub/releases/download/v${STRIPE_STUB_VERSION}/stripestub_${STRIPE_STUB_VERSION}_linux_amd64.tar.gz" -o "stripestubs/stripestub_${STRIPE_STUB_VERSION}_linux_amd64.tar.gz"
+      tar -zxf "stripestubs/stripestub_${STRIPE_STUB_VERSION}_linux_amd64.tar.gz" -C "stripestubs/stripestub_${STRIPE_STUB_VERSION}/"
+    fi
+  - |
+    stripestubs/stripestub_${STRIPE_STUB_VERSION}/stripestub -port ${STRIPE_STUB_PORT} &
+    STRIPE_STUB_PID=$!
```

Comment:
This looks daunting, but it's simple: we fetch a stripestub release, unpack it, and run it. This also does some caching to make setup a bit faster.